circuit-conversation-list.component.js

```diff
@@ -18,14 +18,15 @@ export default Vue.component('circuitConversationList', {
   },
   template: `
     <div>
-      <select class="select-wide" :size="size">
-        <option v-for="conversation in conversations" @click="conversationSelected(conversation)">{{ conversation.topic || conversation.topicPlaceholder }}</option>
+      <select class="select-wide" :size="size"  @change="conversationSelected(selected)" v-model="selected">
```

Comment:
can't you just use "conversation" here instead of using a "selected" variable?

Reply:
Thats what I did at the start, that works fine in chrome. For some reason it complains about conversation being undefined in IE.